admin/app/templates/govau_template.html

```diff
@@ -53,6 +53,9 @@
               <span>{{ global_header_text|default('GOV.AU') }}</span>
               <span class="header-badge">alpha</span>
             </a>
+            <a href="#proposition-menu" id="open-navigation">Menu 
+              <img src="{{ asset_path }}images/navigation-open.svg" alt="Open navigation"/>
+            </a>
           </div>
           {% block inside_header %}{% endblock %}
         </div>
```

Comment:
Not sure if you had planned to do it in this PR but can we have the home page look like the following?

![image](https://user-images.githubusercontent.com/626664/47543886-1643cd80-d930-11e8-8425-1bcfd1befc82.png)

Reply:
Made some changes in this area. Let me know what you think @jonathaningram